### ISSUE TITLE
Can not resize the pane when opening float window

### ISSUE THREAD

**Issue by dinhhuy258** [aserowy/tmux.nvim#53]
The plugin does not work when opening a float window.
You can find the details in the following video.

https://user-images.githubusercontent.com/17776979/146398989-ad14414e-a5a3-4479-8620-f2ccaed77e69.mov

**Comment by aserowy** [comment]
Hi,

do you mean a floating window?
I do not get what you are trying to show :D sry

Kind regards
Alexander

**Comment by dinhhuy258** [comment]
Yes, As you have seen in the demo. When the floating window is not visible I can change the size of the pane. But when the floating window is displayed, I can't resize the pane anymore.

**Comment by aserowy** [comment]
heho,

this behavior is currently not supported. If you wish to enable resizing for floating windows, im happy to accept a pr.

Kind regards
